### ISSUE TITLE
[Bug] There is no link to the RSS feed in the specified situation

### ISSUE THREAD

**Issue by jojoPL1987** [raingart/Nova-YouTube-extension#60]
If the "Add RSS feed link" function is enabled in the script settings, there is no button with an RSS link on the feed if it does not have a large banner. For example, one Polish YT channel with 1.03 million subscriptions. So, as you can see, even among millions of YT channels, there are those that do not have a large banner set. Which results in not displaying graphics with a link to the RSS feed.

https://www.youtube.com/@Ziemniak/videos

 - Browser: Vivaldi 5.7.2921.60 (Stable channel) (64-bit) 
 - Violentmonkey 2.14.0
 - Nova 0.39.0

**Comment by raingart** [comment]
I knew that ordinary users might not have a banner with links. There is no point in subscribing to them.
But that the big channels didn’t have it is amazing. I thought all of them in the training manuals "the way of a popular blogger" must have, next to begging for a subscription and like, there should be links to all their social networks.
But the path of the samurai is rarely chosen by anyone. I think he just forgot to add them with the banner.

There are two location options. Which one do you like more?

**near the  channel-name**
![Screenshot_1](https://user-images.githubusercontent.com/13064767/222687035-ffea0ffd-3605-4abc-86e9-8b28af38a991.png)

**near the  subscribe button**
![Screenshot_49](https://user-images.githubusercontent.com/13064767/222687037-b238719e-475a-40a9-925e-2ba9779bda26.png)

**Comment by raingart** [comment]
Yep, this is an oversight of the author. In the description, the author did not forget to add links to social networks. Most likely this is carelessness on his part than an exception to the rule.

![Screenshot_2](https://user-images.githubusercontent.com/13064767/222691277-6aaf5b3e-6e2f-4575-9d26-7b5719178071.png)

**Comment by jojoPL1987** [comment]
> 

For me, it's better next to the "Subscribe" button.

**Comment by raingart** [comment]
ok, when will i add it in the next release

**Comment by raingart** [comment]
If you want to apply now replace this code
```
 // if ((channelName_ = document.body.querySelector('#channel-handle')?.textContent)
                  //    && channelName_.startsWith('@')
                  // ) {
                  //    channelName = channelName_.substring(1);
                  // }
```
 by that
```
if (document.body.querySelector('#header a.banner-visible-area[hidden]')) {
                    container = document.body.querySelector('#channel-header #channel-name'); // #1
                    //container = document.body.querySelector('#inner-header-container #buttons'); #2
                  }
```
